assets/js/script.js

```diff
@@ -0,0 +1,107 @@
+import { UI } from "./ui.js";
+const http = new Http();
+
+const fetchDataFromApi = {
```

Comment:
You don't seem to have anything more stored in this object. Maybe the function itself could be stored on the top level? 😃